samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -111,28 +87,51 @@ private void OnEnable()
         }
     }
 
-    private void PopulateSampleSceneList()
+    public void SetPipelineText(string text)
     {
-        SceneDropdown.options.Clear();
+        PipelineModeText = text;
+    }
 
-#if !(USE_OPENXR_PACKAGE)
-        if (SceneList.Contains("VR-Sample")){ 
-             SceneList.Remove("VR-Sample");
-        }
-#else
-        if (!SceneList.Contains("VR-Sample")){ 
-             SceneList.Add("VR-Sample");
-        }
-#endif
-        SceneDropdown.AddOptions(SceneList);
-        AddScene();
+    public void SetSceneText(string text)
```

Comment:
Just be sure to re-hookup any refs that might break. I believe you can change the editor to `debug` so the ref doesn't break

```suggestion
    public void SetNextSceneName(string text)
```